docs/sources/panels-visualizations/visualizations/bar-gauge/index.md

```diff
@@ -67,6 +67,26 @@ Choose a display mode.
 - **Retro LCD -** The gauge is split into small cells that are lit or unlit.
 - **Basic -** Single color based on the matching threshold.
 
+### Value display
+
+Choose a value display mode.
+
+- **Value color -** Value color is determined by value.
+- **Text color -** Value color is default text color.
+- **Hidden -** Values are hidden.
+
+### Field label placement
```

Comment:
I think name is better term than field label. In the sister panel Stat we refer to it as Name , and in settings we refer to it as "Display name" , never label.

So think "Name placement" as the option name is best